tests/pr_test.go

```diff
@@ -2,38 +2,136 @@
 package test
 
 import (
+	"fmt"
+	"io/fs"
+	"path/filepath"
+	"strings"
 	"testing"
 
 	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
 	"github.com/terraform-ibm-modules/ibmcloud-terratest-wrapper/testhelper"
+	"github.com/terraform-ibm-modules/ibmcloud-terratest-wrapper/testschematic"
 )
 
-const namespaceDir = "examples/namespace"
+const solutionStandardDir = "solutions/standard"
+const resourceGroup = "geretain-test-icr"
```

Comment:
Did you manually create this in the account? I don't see it in https://github.ibm.com/GoldenEye/ge-dev-account-management ? Any permenent resource in our account should be created in that repo as we use terraform to manage them

Reply:
Raised the PR -> https://github.ibm.com/GoldenEye/ge-dev-account-management/pull/564